### PR TITLE
Add completion handling to AdSense wizard and add Google Analytics wizard

### LINKED ISSUE #135: Advertising Wizard: "Finished Setup"
As @claudiulodro [noted](https://github.com/Automattic/newspack-plugin/pull/128#discussion_r298224563):

> After completing the set up, I'm dropped back on this page which continues telling me to set up AdSense. The 'Back to checklist' link reads like a 'Cancel' IMO. Would it make things clearer to also have an "I've finished setting up AdSense" button underneath the handoff button? It would take you to the checklist, and (in the future) it would mark the task complete in the checklist.

e.g.
localhost_wp-admin_admin php_page=newspack-google-adsense-wizard

### PR DESCRIPTION
**Pull request by claudiulodro**
### All Submissions:

* [x] Have you followed the [Newspack Contributing guideline](https://github.com/Automattic/newspack-plugin/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) and [VIP Go coding standards](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Closes #135  .

This PR has a few related changes. The changes are as follows:
- Removed duplicate/dummy `class-sitekit-configuration-manager.php` and updated things to use the real configuration manager (`class-site-kit-configuration-manager.php`).
- Added a feature when the user comes to the AdSense wizard that will check if AdSense if configured, and if it is configured, it will render that AdSense setup is complete and mark it complete on the checklist. (screenshot below)
- Implemented a Google Analytics wizard based on the AdSense wizard. It has basically the same UI as the AdSense Wizard. It's linked directly from the Analytics dashboard card. This is slightly different than what the designs called for, and we can update it when we add another supported analytics provider. When the wizard is complete, you should see the Analytics card marked as 'complete' (screenshot below)

### Screenshots
localhost_wp-admin_admin php_page=newspack-google-adsense-wizard (3)
localhost_wp-admin_admin php_page=newspack-google-adsense-wizard (2)
localhost_wp-admin_admin php_page=newspack (1)

### How to test the changes in this Pull Request:

If you have Site Kit working on your local environment, you should be able to build things like normal and simply test the 2 wizards. 

If you don't have Site Kit working on your local environment, you can change [this line](https://github.com/Automattic/newspack-plugin/blob/master/includes/configuration_managers/class-site-kit-configuration-manager.php#L62) to `return true;` to simulate that a module has been successfully configured.

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?

### CODE REVIEW

**Comment by sonjaleix** [comment]
Google Analytics wizard's looking great! Approved from my side.

**Comment by claudiulodro** [comment]
The handoff URL issue was a bug. It should be fixed in https://github.com/Automattic/newspack-plugin/pull/158/commits/dfa3744b21328757dbcc110c97735161f98d122c, and you should get handed off to the appropriate settings section now.

File: assets/wizards/googleAnalytics/index.js [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+			} )
+			.catch( error => {
+				this.setError( error );
+			} );
+	}
+
+	/**
+	 * Mark this wizard as complete.
+	 */
+	markWizardComplete() {
```
[the remark this answers is not in the document]
Comment (claudiulodro) [reply assets/wizards/googleAnalytics/index.js]: I think we could definitely move the `markWizardComplete` method into `withWizard` somehow and simplify that part. I don't think we can really generalize when things are marked complete though. For example, this wizard is marked complete when returning to the wizard and Analytics is configured. Another wizard is marked complete when hitting Save on the last screen of a series of screens.

**Comment by jeffersonrabb** [comment]
> The handoff URL issue was a bug. It should be fixed in dfa3744, and you should get handed off to the appropriate settings section now.

Works great now.

File: assets/wizards/googleAnalytics/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (jeffersonrabb) [reply assets/wizards/googleAnalytics/index.js]: Yes, agreed. We might handle it like `setError`, where `withWizard` provides the function as a prop, and the Wizard Component takes responsibility for when to call it.

**Pull request merged**